file_transfer_agent.go

```diff
@@ -697,31 +697,19 @@ func (sfa *snowflakeFileTransferAgent) upload(
 }
 
 func (sfa *snowflakeFileTransferAgent) download(
-	largeFileMetadata []*fileMetadata,
-	smallFileMetadata []*fileMetadata) error {
+	fileMetadata []*fileMetadata) error {
```

Comment:
Can we now join this two lines?

Reply:
yeah I changed the L197 to only take in the small metadata since large metadata is unused now.